### PR TITLE
exluded characters in random string generator

### PR DESCRIPTION
**Pull request by musinit**

### CODE REVIEW

File: faker.go [hunk 1]
```diff
@@ -1048,13 +1053,32 @@ func extractNumberFromText(text string) (int, error) {
...
+			set[s] = struct{}{}
+		}
+	}
+
+	counter := 0
 	for i := 0; i < n; {
 		randRune := rune(rand.Intn(int(lang.end-lang.start)) + int(lang.start))
+		for slice.ContainsRune(set, randRune) {
+			if counter++; counter >= maxGenerateStringRetries {
+				log.Fatal("Can't generate random sequence with exluded letters")
```
Comment (bxcodec) [inline faker.go:1068]: Can we return an error here? 
Using fatal is quite dangerous, so I think, we just return the error, and let the user decide to do what they need to do, like calling fatal, or panic, or just log it?

Comment (bxcodec) [reply faker.go:1068]: The error message is maybe something related to max-retries?

Comment (musinit) [reply faker.go:1068]: Yes, sure. I will return an error and the error will indicate info about exhaustion of max-retries

**Comment by musinit** [comment]
@bxcodec The test coverage of internet package decreased significantly in this PR, mainly because I don't know how better to cover cases, where success of randomString method depends on tiny probability that we each time for 1000000 retries we will point on 6 excluded elements from 57

File: internet.go [hunk 2]
```diff
@@ -94,59 +106,80 @@ func MacAddress() string {
...
-	return internet.username(), nil
+	return internet.username()
 }
 
 // Username get username randomly in string
 func Username() string {
 	return singleFakeData(UserNameTag, func() interface{} {
 		i := Internet{}
-		return i.username()
+		u, _ := i.username()
```
Comment (bxcodec) [inline internet.go:181]: This one is not consistent with the others. I see the others is calling panic, but this one is ignored. Let's keep it consistent, either we call panic or ignore it.

Comment (musinit) [reply internet.go]: Fixed!

**Pull request merged**